src/test/java/tests/UsersTest.java

```diff
@@ -124,4 +125,98 @@ public void youngerThanParameterTest() {
                 () -> Assertions.assertFalse(usersResponse.getBody().isEmpty(), "Response body does not contain user"),
                 () -> Assertions.assertTrue(usersResponse.getBody().stream().allMatch(user -> user.getAge() < 19), "User age is not younger than than target age"));
     }
+
+    @Test
+    public void putUserTest() {
+        ResponseEntity<List<User>> usersResponse = client.getUsers();
+        User existingUser = usersResponse.getBody().get(0);
+        User userToChange = User.builder()
+                .name(existingUser.getName())
+                .sex(existingUser.getSex())
+                .age(existingUser.getAge())
+                .zipCode(existingUser.getZipCode())
+                .build();
+
+        User userUpdateValues = User.builder()
+                .age(30)
+                .name(RandomStringUtils.randomAlphabetic(10))
+                .sex(Gender.FEMALE)
+                .zipCode(zipcode)
+                .build();
+
+        UpdateUser updateUserBody = UpdateUser.builder()
```

Comment:
I think there is no need to use builder where using a constructor with args is shorter. Ex.:
`client.putUser(new UserToUpdate(userToChange, userUpdateValues));`